match-pattern/capsule.bxb

```diff
@@ -5,11 +5,13 @@ capsule {
   targets {
     target (bixby-mobile-en-US)
   }
-  runtime-version (2)
+  runtime-version (2) {
```

Comment:
let's make sure we bring up the runtime version to the latest (9) for all sample capsules going forward.

Reply:
can you re-run the tests after the update?